src/ralph/backends/data/base.py

```diff
@@ -79,6 +77,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class Loggable:
+    """A class with a `logger` property."""
+
+    @functools.cached_property
```

Comment:
nit: We could import `cached_property` from `functools`
```suggestion
    @cached_property
```

Reply:
Well spotted) Thanks) I have updated [this](https://github.com/openfun/ralph/pull/521/files#diff-abc4154695f25bd1399562f7c5e9f27fe9fba44ce29e46d606b46f1871b932a6R89) in PR #521 - could we postpone the change here?

Reply:
After thought, I think it would be better to make this change here) will rebase #521 )

Reply:
Alright, as you prefer! :raised_hands: